### PR TITLE
[#6786] restyle main wallet screen

### LINKED ISSUE #6786: Wallet main screen restyle
Restyle the wallet main screen so resembles the main screen in the new wallet flow.

Figma Wallet Setup
https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction?node-id=0%3A1

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### PR DESCRIPTION
**Pull request by bhauman**
fixes #6786  Wallet main screen re-style for new wallet flow

### Summary:

This re-styles the wallet main screen according to for `main-01` in the following figma https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction.

### Testing notes:

Check that the screens match the figma and that they function as intended.

### Steps to test:
- Open Status
- Navigate to wallet
- compare the screen to `main-01` on the figma linked above

Thanks!

status: ready

<blockquote><div> Figma</div><div><strong><a href="https://www.figma.com/file/SIeZPHvrX7KUwEBW8VQct8C5/WIP_New-send-transaction">Figma</a></strong></div><div>Created with Figma</div></blockquote>

### CODE REVIEW

**Comment by bhauman** [comment]
These changes are purely cosmetic.

Android screen shot:
android_screen

IOS Screen Shot:
simulator screen shot - iphone 7 - 2018-11-19 at 21 57 09

**Comment by chadyj** [comment]
ping @denis-sharypin!

**Review by goranjovic — APPROVED** [review]

**Comment by asemiankevich** [comment]
@flexsurfer @yenda please add your reviews asap

**Comment by asemiankevich** [comment]
@errorists sorry for so many pings, i am forced to ask you while Denis is out. Would you mind to check this real quick too? Thanks!

**Comment by asemiankevich** [comment]
@antdanchenko @yevh-berdnyk @hetvart:
- `Receive transaction` is replaced with `Deposit`
- `Send transaction` is replaced with `Send`

please update tests.

**Comment by hetvart** [comment]
> @antdanchenko @yevh-berdnyk @hetvart:
> 
> * `Receive transaction` is replaced with `Deposit`
> * `Send transaction` is replaced with `Send`
> 
> please update tests.

I've just downloaded the e2e `*.apk` and tried to run tests with the buttons from `develop` branch and it works as expected. We used `accessibility_id` for these locators so no need to change them. (https://status-im-prs.ams3.digitaloceanspaces.com/StatusIm-181121-165435-955356-e2e.apk)

Thanks to update us! 

<blockquote></blockquote>

File: src/status_im/ui/screens/wallet/main/styles.cljs [hunk 1]
```diff
@@ -100,9 +99,17 @@
...
 (def asset-section
   {:flex             1
-   :padding-top      16
+   :padding-top      5
+   :padding-bottom   20
    :background-color colors/white})
 
+(def asset-section-header
+  {:font-size     15
+   :color         "#939ba1"
```
Comment (yenda) [inline src/status_im/ui/screens/wallet/main/styles.cljs]: colors/gray

**Review by yenda — CHANGES_REQUESTED** [review]

**Comment by asemiankevich** [comment]
screens look good

image

image

**Review by yenda — APPROVED** [review]

**Comment by bhauman** [comment]
@asemiankevich is this all set to go?

**Comment by asemiankevich** [comment]
yess)

**Comment by oskarth** [comment]
@goranjovic why did we rename 'Receive' to 'Deposit'?

**Pull request merged**